be/src/vec/core/field.h

```diff
@@ -339,6 +339,13 @@ class DecimalField {
   * Used to represent a single value of one of several types in memory.
   * Warning! Prefer to use chunks of columns instead of single values. See Column.h
   */
+
+struct DetailedTypeInfo {
```

Comment:
Is there any existed class can be reused? Adding a new type class will add more complexity.

Reply:
remove this structure